.github/workflows/gradle.yml

```diff
@@ -16,7 +16,7 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: windows-2022
```

Comment:
You still want to make sure those ./gradlew permissions are correct since this code should run on any OS

Reply:
`git update-index --chmod=+x foo.sh`
https://stackoverflow.com/questions/21691202/how-to-create-file-execute-mode-permissions-in-git-on-windows

Reply:
Oops. Also need to make sure that the gradle-wrapper.jar is present. Did we get bitten by .gitignore ?

Reply:
Yes. There's a *.jar line in .gitignore. That's going to confuse the build as well.

Reply:
Got the permissions for gradlew (not gradlew.bat) fixed and confirmed that the gradle build runs in both ubuntu and windows containers.